dlt/destinations/job_client_impl.py

```diff
@@ -232,9 +233,12 @@ def _null_to_bool(v: str) -> bool:
                 return True
             raise ValueError(v)
 
+        fields = ["column_name", "data_type", "is_nullable"]
```

Comment:
OK! I just wonder if the INFORMATION_SCHEMA table is defined in itself and we could pull those columns. but indeed - your method is faster. let's keep it